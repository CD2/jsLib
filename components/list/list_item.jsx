import React from 'react'
import PropTypes from 'prop-types'

export default class ListItem extends React.Component {

  static propTypes = {
    children: PropTypes.node,
    className: PropTypes.string,
  }

  render() {
    const { children, className=`` } = this.props
    return (
      <div className={`list_item ${className}`}>
        { children }
      </div>
    )
  }

}
