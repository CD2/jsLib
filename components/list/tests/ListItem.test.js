/* eslint-disable react/jsx-filename-extension */
import React from "react"
import { ListItem } from "../list_item"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

describe(`<ListItem />`, () => {
  const view = shallow(<ListItem />)

  it(`snapshot`, () => expect(toJson(view)).toMatchSnapshot())
})
