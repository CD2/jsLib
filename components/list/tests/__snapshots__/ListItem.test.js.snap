// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> snapshot 1`] = `
<div
  className="list_item "
/>
`;

exports[`snapshot 1`] = `
<div
  className="list_item "
/>
`;
